src/components/Partners.astro

```diff
@@ -0,0 +1,101 @@
+<head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@600&display=swap" rel="stylesheet">
+</head>
```

Comment:
Move it to `<head>` in [Layout.astro](https://github.com/tul-ftims-students-council/Delta/blob/master/src/layouts/Layout.astro)